src/Propel/Generator/Config/GeneratorConfig.php

```diff
@@ -280,9 +280,9 @@ public function getBehaviorLocator()
      *
      * @return object
      */
-    private function getInstance($className, $arguments = null, $interfaceName = null)
+    private function getInstance(?string $className, $arguments = null, $interfaceName = null)
     {
-        if (!class_exists($className)) {
+        if (!$className || !class_exists($className)) {
```

Comment:
mb we can change exception message accordingly?

Reply:
I've fixed it properly now - 26b0cbe30dedbf5472d0761c1e40dc0a946447e5